### PR TITLE
common: check I>1/2 in setB

### PR DESCRIPTION
**Pull request by sotirova**
Currently working with the ground level of spin-1/2 nuclei fails as it gives 0 denominator in the [Hamiltonian](https://github.com/OxfordIonTrapGroup/atomic_physics/blob/b32491c9877e9c758a4613207005e822de82f393/atomic_physics/common.py#L369) in `setB`. Hence, we need to check that both `J > 1/2` and `I > 1/2`.

### CODE REVIEW

**Comment by hartytp** [comment]
Thanks.

Before merging we should add a test. @sotirova could you have a go at that please? (see e.g. `ionics_fits` for an example of how to add a `pytest` `poe` task)

File: atomic_physics/tests/test_spin_half.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from atomic_physics.utils import Lande_g
+import numpy as np
+import scipy.constants as ct
+
+
+def _gF(F, J, I, gJ):
+    return gJ * (F * (F + 1) + J * (J + 1) - I * (I + 1)) / (2 * F * (F + 1))
+
+
+class TestSpinHalf(unittest.TestCase):
```
Comment (hartytp) [inline atomic_physics/tests/test_spin_half.py:13]: can we use `pytest` here (see how it's done in `ionics_fits`)

Comment (hartytp) [reply atomic_physics/tests/test_spin_half.py:13]: (and make these functions rather than classes)

Comment (sotirova) [reply atomic_physics/tests/test_spin_half.py:13]: Yes - though isn't it best to stay consistent with the current tests in this package? Or is there a plan to change those as well?

Comment (hartytp) [reply atomic_physics/tests/test_spin_half.py:13]: I'd forgotten we had tests. In that case, yes, stick with this style. Thanks for the correction!

Comment (sotirova) [reply atomic_physics/tests/test_spin_half.py:13]: Ah - cool! May I merge this now?

**Comment by hartytp** [comment]
yes, thanks!

**Review by hartytp — APPROVED** [review]

**Pull request merged**